lib/raidenclient/RaidenClient.ts

```diff
@@ -101,11 +108,27 @@ class RaidenClient extends BaseClient {
       this.logger.error(`can't init raiden. raiden is disabled`);
       return;
     }
+    // associate the client with all currencies that have a contract address
+    await this.setCurrencies();
     // set status as disconnected until we can verify the connection
     this.setStatus(ClientStatus.Disconnected);
     await this.verifyConnection();
   }
 
+  private setCurrencies = async () => {
+    try {
+      const currencies = await this.repository.getCurrencies();
+      const raidenCurrencies = currencies.filter((currency) => {
+        const tokenAddress = currency.getDataValue('tokenAddress');
+        if (tokenAddress) {
+          this.currencies[currency.getDataValue('id')] = tokenAddress;
+        }
+      });
+    } catch (e) {
+      this.logger.error(`failed to set currencies for Raiden: ${e}`);
```

Comment:
If you make this `this.logger.error('failed to set currencies for Raiden', e);` the `error` method will log the exception message and stack trace if applicable.